buildenv/builder.pl

```diff
@@ -365,7 +365,15 @@ sub addPkg {
                     $package->{"outputs_to_install"} = $package->{"outputs-to-install"};
                 }
                 foreach my $output (keys %{$package->{"outputs"}}) {
-                    next unless grep { $_ eq $output } @{$package->{"outputs_to_install"}};
+                    # Unfortunately, due to pkgdb limitations in the 1.0 release we
```

Comment:
nit: it was a deliberate decision, and we haven't decided to change the default. We may decide to change the default as part of https://github.com/flox/flox/issues/1363